o8g/Scripts/autoscripts.py

```diff
@@ -1895,9 +1898,6 @@ def per(Autoscript, card = None, count = 0, targetCards = None, notification = N
       else: #If we're not looking for a particular target, then we check for everything else.
          debugNotify("Doing no table lookup") # Debug.
          if per.group(3) == 'X': multiplier = count # Probably not needed and the next elif can handle alone anyway.
-         elif count: multiplier = num(count) * chkPlayer(Autoscript, card.controller, False) # All non-special-rules per<somcething> requests use this formula.
-                                                                                              # Usually there is a count sent to this function (eg, number of favour purchased) with which to multiply the end result with
-                                                                                              # and some cards may only work when a rival owns or does something.

```

Comment:
I removed this code because I do not think it is relevant for the Doomtown (maybe was used in ANR or SW). Thing is, that it is preventing usage of redirected autoscripts (after $$) such as (Put1Bounty-perProperty{}). 
This is because after the redirect, there is always some count, so the removed branch executes instead of doing the perProperty (or perMarker and others in other elif branches below).
Also, I checked all Card scripts and only per scripts after redirects are using either perX (how it should be used), or per(Target|Host|Every). So nothing should be broken.